src/ini.cr

```diff
@@ -19,4 +19,21 @@ class INI
     end
     ini
   end
+
+  # Generates an INI-style configuration from a given hash.
+  #
+  # ```
+  # INI.build({"foo" => {"a" => "1"}}, " ") # => "[foo]\na = 1\n\n"
+  # ```
+  def self.build(ini : Hash(String, Hash(String, String)), space : String = "") : String
+    io = IO::Memory.new
+    ini.each do |section, contents|
+      io << '[' << section << "]\n"
+      contents.each do |key, value|
+        io << key << space << '=' << space << value << "\n"
+      end
+      io << "\n"
```

Comment:
Last one, promise, Crystal actually interprets `\n` within single quotes since they're used to distinguish the type, not whether escape sequences are interpreted ;)